tensorflow_lite_support/tools/ci_build/builds/build_ios_framework.sh

```diff
@@ -58,140 +60,51 @@ if [[ -z "${DEST_DIR+x}" || "${DEST_DIR}" == ${TFLS_ROOT_DIR}* ]]; then
   exit 1
 fi
 
-
-# Builds the C API framework for the specified framework for iOS.
-function build_c_api_framework {
-  "${BAZEL}" build -c opt --config=ios_fat \
-    //tensorflow_lite_support/ios:$1C_framework
+# get_output_file_path takes one bazel target label as an argument, and prints
+# the path of the first output file of the specified target.
+function get_output_file_path {
+  local STARLARK_FILE="${OUTPUT_TMPDIR}/print_output_file.starlark"
```

Comment:
If the `OUTPUT_TMPDIR` is only used for the bazel cquery, shouldn't it be defined within the `get_output_file_path` function in the first place?

Then, the directory should ideally be cleaned up at the end of `get_output_file_path`.

Reply:
Fixed